src/jit/lclvars.cpp

```diff
@@ -2847,6 +2847,15 @@ void Compiler::lvaSortByRefCount()
             lvaSetVarDoNotEnregister(lclNum DEBUGARG(DNER_PinningRef));
 #endif
         }
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+        else if (opts.MinOpts() && !JitConfig.JitMinOptsTrackGCrefs())
+        {
+            if (varTypeIsGC(varDsc->TypeGet()))
+            {
```

Comment:
minor: you may want all the conditions on the "else if" line above. Otherwise, if someone adds yet another "else if" case below, then it wouldn't get hit for the minopts & !JitMinOptsTrackGCrefs() case if the var isn't GC.